blog-server-api/src/main.rs

```diff
@@ -34,13 +34,41 @@ async fn main() -> screw_components::dyn_result::DResult<()> {
 
 pub async fn init_db() -> rbatis::RBatis {
     let rb = rbatis::RBatis::new();
-    rb.init(rbdc_pg::driver::PgDriver {}, PG_URL).unwrap();
-
-    let sql = std::fs::read_to_string("./table_pg.sql").unwrap();
-    rb.exec(&sql, vec![]).await.expect("DB migration failed");
+    rb.init(rbdc_pg::driver::PgDriver {}, PG_URL)
+        .expect("DB init failed");
+    migrate_db(&rb).await.expect("DB migration failed");
     return rb;
 }
 
+async fn migrate_db(rb: &rbatis::RBatis) -> Result<(), Box<dyn std::error::Error>> {
+    let sql = std::fs::read_to_string("./table_pg.sql")?;
+    rb.exec(&sql, vec![]).await?;
+
+    let posts: Vec<blog_server_services::traits::post_service::Post> =
```

Comment:
We also can add some sort of flag to run this only once

Reply:
done